Gopkg.toml

```diff
@@ -44,7 +44,7 @@
 
 [[constraint]]
   name = "github.com/kubernetes-csi/csi-lib-utils"
-  version = "0.1.0"
+  version = "0.3.0"
```

Comment:
do we want to constrain the version here or just use the latest?

Reply:
I'm for "use the latest" and the constraint does allow that because it means ">= 0.3.0". So this looks right to me.